styles/base/inputs/_select.scss

```diff
@@ -47,14 +46,23 @@
 }
 
 .sprk-b-Select[disabled] {
-  border-color: $sprk-select-disabled-border-color;
   background-color: $sprk-select-disabled-background-color;
   color: $sprk-select-disabled-color;
+  box-shadow: $sprk-text-input-disabled-box-shadow;
+  cursor: not-allowed;
```

Comment:
We made a variable for this in the other instance.

Reply:
Yeah, anything we can make a var now will save us in the long run.